packages/hash/frontend/src/pages/api/ory/[...paths].api.ts

```diff
@@ -0,0 +1,12 @@
+// @ory/integrations offers a package for integrating with NextJS.
+import { config, createApiHandler } from "@ory/integrations/next-edge";
+
+// We need to export the config.
+export { config };
+
+const publicUrl = process.env.ORY_KATROS_PUBLIC_URL;
```

Comment:
Kratos not katros

Reply:
Addressed in https://github.com/hashintel/hash/pull/889/commits/7d49886a39cbbd6e062ffa84c7aa9d3abda388d1